components/org.wso2.carbon.identity.oauth.endpoint/src/main/java/org/wso2/carbon/identity/oauth/endpoint/util/EndpointUtil.java

```diff
@@ -1773,4 +1776,8 @@ public static boolean isExternalConsentPageEnabledForSP(ServiceProvider serviceP
         }
         return isEnabled;
     }
+
+    public static boolean isConsentPageRedirectParamsAllowed() {
+        return FileBasedConfigurationBuilder.getInstance().isConsentPageRedirectParamsAllowed();
```

Comment:
Add a new line after the method signature.

Reply:
Resolved by https://github.com/wso2-extensions/identity-inbound-auth-oauth/pull/2075/commits/797a2d4fb1cd55a1395d0d30d6852a97b9e9edcf